code/modules/paperwork/faxmachine.dm

```diff
@@ -169,9 +181,19 @@ var/list/sent_faxes = list()	//cache for faxes that have been sent by the admins
 
 	updateUsrDialog()
 
-/obj/machinery/photocopier/faxmachine/proc/sendfax(var/destination)
+// Only run this in a spawn()!
+/obj/machinery/photocopier/faxmachine/proc/process_cooldown()
```

Comment:
Amended in fb57de624b267af4bf3366a7d38409cc604060c2, but used a slightly different approach to determine the remaining cooldown time.